tests/lib/cli.js

```diff
@@ -570,21 +570,21 @@ describe("cli", () => {
 
                 it(`should only print error with configType:${configType}`, async () => {
                     const filePath = getFixturePath("single-quoted.js");
-                    const cliArgs = `--no-ignore --quiet  -f compact --rule 'quotes: [2, double]' --rule 'no-unused-vars: 1' ${filePath}`;
+                    const cliArgs = `--no-ignore --quiet  -f stylish --rule 'quotes: [2, double]' --rule 'no-unused-vars: 1' ${filePath}`;
 
                     await cli.execute(cliArgs, null, useFlatConfig);
 
                     sinon.assert.calledOnce(log.info);
 
                     const formattedOutput = log.info.firstCall.args[0];
 
-                    assert.include(formattedOutput, "Error");
-                    assert.notInclude(formattedOutput, "Warning");
+                    assert.include(formattedOutput, "1 error");
+                    assert.include(formattedOutput, "0 warnings");
```

Comment:
...

Now, without the `--quiet` flag, the output would be:

```
C:\Users\milos\AppData\Local\Temp\eslint\fixtures\single-quoted.js
  1:1   warning  'console' is not defined      no-undef
  1:13  error    Strings must use doublequote  quotes

✖ 2 problems (1 error, 1 warning)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

However, the test is still passing because it finds `0 warnings` in `1 error and 0 warnings potentially fixable`.

To make sure that the test works as intended, it might be better to check something like this:

```suggestion
                    assert.include(formattedOutput, "(1 error, 0 warnings)");
```